### DISCUSSION TITLE
Can't access some directories from webradios since last update (28/08/22)

### DISCUSSION THREAD

**Discussion by bTronics** [rern/rAudio#1002] [Fixed Issues]
Hello, 

Since the last update, I can't access Every last directory from the webradios tab.
Is there a workaround?

Greetz

**Comment by rern** [comment]
Fixed with 20220902 update.

**Comment by bTronics** [comment]
Hello,
Thank you for the update, but i have still a problem. Now I can access the directories, but the radio stations are not visible.

**Reply by rern** [reply]
Please post the Web Radio name and URL.

**Reply by bTronics** [reply]
With the update 20220828 I couldn't access every last directory. With the update 20220902 this was fixed but all the last directories seems to be empty. Before the updates all these directories were full of radio stations. With my laptop and 'putty' I looked in: /srv/http/data/webradios. Then I can see all my radio stations in the directories. So the stations are present, but in raudio i see just an empty directory.
Best regards.

**Reply by rern** [reply]
The Web Radio directory has been changed from `webradios` to `webradio` by updates since 20220805 to the latest. If it's not a typo, rename the directory with commands from the update script will fix your issue:
```sh
dirdata=/srv/http/data
if [[ -e $dirdata/webradios ]]; then
	mv $dirdata/webradio{s,}
	mv $dirdata/{webradiosimg,webradio/img}
fi
```
By the way, how is the `webradios` still there after updates?

**Reply by bTronics** [reply]
I don't know what happens. I guess something went wrong when i copied my radios from an earlier installation. Now I made a complete new installation without copying the radio stations. Now I see it's not longer possible to make a new directory within a directory. I think this is why I couldn't see my radio stations because I copied them from an older installation and there I created directories within directories.
Is this feature not implemented anymore?
